src/@ionic-native/plugins/nfc/index.ts

```diff
@@ -49,6 +49,21 @@ declare let window: any;
 */
 @Injectable()
 export class NFC extends IonicNativePlugin {
+  /**
+   * Starts the NFCNDEFReaderSession allowing iOS to scan NFC tags.
+   * @param onSuccess
+   * @param onFailure
+   * @returns {Observable<any>}
+   */
+  @Cordova({
+    observable: true,
+    successIndex: 0,
+    errorIndex: 3,
+    clearFunction: 'invalidateSession',
```

Comment:
will this work although the function is not defined in the wrapper?

Reply:
Apparently, yes. @stormwin said the ionic wrapper would be smart enough to call this function.